tpm2/constants.go

```diff
@@ -136,6 +142,23 @@ const (
 	StartupState
 )
 
+// ECCCurve identifies specific ECC curves.
```

Comment:
ECCCurve == "Elliptic Curve Cryptography Curve", which seems redundant to me. How about just EllipticCurve?

Reply:
Done.
I was trying to keep this close to name in the spec, which is TPM_ECC_CURVE.
But agree, it's more awkward and not necessary